main.py

```diff
@@ -95,11 +107,37 @@ def init_schedule():
     latest_schedule.update(queue)
 
 
+async def init_dataset_tracker() -> asyncio.Task:
+    """Initializes the dataset tracker and runs the subscriber.
+    
+    This should be called after loading config.
+    """
+    global dataset_tracker  # pylint: disable=global-statement
+    maxlen = configs["dataset_tracker"].get("maxlen", 1 << 16)
```

Comment:
Is there any reason to set the default maxlen to "1 << 16"?

Reply:
I just wanted to set it to a power of 2, and I thought it should be greater than ~50,000 to store enough modifications.
I'm not sure yet how the size affects the performance.